translations/fi.ts

```diff
@@ -2930,47 +2930,47 @@ Hide formatting characters:
     </message>
     <message>
         <source>If disabled, mute notification sounds when status is Busy (but still show a notification)</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Jos se ei ole käytössä, mykistää ilmoitusäänet, kun tila on Varattu (mutta näytä silti ilmoitus)</translation>
     </message>
     <message>
         <source>Use desktop notifications, e.g. in a notification center or at the system tray. Without this, the notification will only be visible as a flashing tray icon.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Käytä työpöytäilmoituksia, esim. ilmoituskeskuksessa tai ilmaisinalueella. Ilman tätä ilmoitus näkyy vain vilkkuvana tarjotinkuvakkeena.</translation>
```

Comment:
There are a few like this that seem a bit odd, though not exactly wrong, but this one is a bit more so.

The tray icon translation is a bit too literal, I think, more referring to a tray used for serving food for example than a 'notification tray' or so.

Could just use 'vilkkuvana kuvakkeena' here.

Reply:
Can you make an account at weblate and make that change after this PR? Everything here is Google Translated. It's almost certainly quite broken in many places, but I figured it'd be better than nothing, and it gives incentives to people to go and fix the translation :).

Reply:
In fact, you can make the change in weblate now. I can easily fix any conflicts that come up.

Reply:
Yeah, figured as much, and that's what I was planning on doing when I have more time and/or energy.  ^^